python/vyos/ifconfig/interface.py

```diff
@@ -929,7 +930,43 @@ def set_dhcpv6(self, enable):
 
             if os.path.isfile(config_file):
                 os.remove(config_file)
-
+    
+    def get_tc_config(self,objectname):
+        # Parse configuration
+        get_tc_cmd = f'tc -j {objectname}'
+        tmp = cmd(get_tc_cmd, shell=True)
+        return json.loads(tmp)
+    
+    def del_tc_qdisc(self,dev,kind,handle):
+        tc_qdisc = self.get_tc_config('qdisc')
+        for rule in tc_qdisc:
+            old_dev = rule['dev']
+            old_handle = rule['handle']
+            old_kind = rule['kind']
+            if old_dev == dev and old_handle == handle and old_kind == kind:
+                if 'root' in rule and rule['root']:
+                    delete_tc_cmd = f'tc  qdisc del dev {dev} handle {handle} root {kind}'
```

Comment:
What is the reason we need `root` and we do not require it in the previous implementation?

Reply:
In fact, this is needed to realize the export image. However, this part is not used because it focuses on migration first